src/view/element.js

```diff
@@ -712,6 +712,48 @@ export default class Element extends Node {
 			( attributes == '' ? '' : ` ${ attributes }` );
 	}
 
+	/**
+	 * Creates element instance from provided viewElementDefinition.
+	 *
+	 * @param {module:engine/view/viewelementdefinition~ViewElementDefinition} viewElementDefinition
+	 * @returns {Element}
+	 */
+	static fromViewDefinition( viewElementDefinition ) {
+		const attributes = {};
+
+		const classes = viewElementDefinition.classes;
+
+		if ( classes ) {
+			attributes.class = Array.isArray( classes ) ? classes.join( ' ' ) : classes;
+		}
+
+		const stylesObject = viewElementDefinition.styles;
+
+		if ( stylesObject ) {
+			attributes.style = toStylesString( stylesObject );
```

Comment:
Do you guys think that we could avoid this stringification? E.g. by setting styles after creating this element (then, view `Element` should then have `setStyles()` method)?

Uncareful stringification is unsafe.